### DISCUSSION TITLE
Support request for NV3041A

### DISCUSSION THREAD

**Discussion by chrismendes-waterloo** [Bodmer/TFT_eSPI#2579] [New processor/display support or capability request]
HI team - would it be possible to add support for NV3041A? There are several screens available in China now that are carrying this chip. ArduinoGFX has support already.

https://github.com/moononournation/Arduino_GFX/blob/master/src/display/Arduino_NV3041A.cpp

Thanks kindly.

**Comment by Bodmer** [comment]
Post links to the sellers web page.

**Reply by saeugetier** [reply]
I recently bought one, which was cheaper than the yellow cheap displays: https://de.aliexpress.com/item/1005006729657546.html

You can find information about that board: https://github.com/profi-max/JC4827W543_4.3inch_ESP32S3_board

**Reply by saeugetier** [reply]
Reason, why the driver is requested: https://github.com/HASwitchPlate/openHASP/issues/713#issue-2262057051

**Comment by 01GOD** [comment]
Sigh...has the NV3041A been added to TFT_eSPI yet? Tried using Squareline Studio with that display today and found out that Squareline Studio sadly seemingly relies on TFT_eSPI library to function with displays. 
If any here recommend a solution or workaround, thanks in advance.

**Comment by chrismendes-waterloo** [comment]
It's supported via arduino_gfx. I have it compiling and working with
squareline.

**Reply by 01GOD** [reply]
> It's supported via arduino_gfx. I have it compiling and working with squareline.
> […](#)
> On Sat, Aug 3, 2024, 11:18 AM 01GOD ***@***.***> wrote: Sigh...has the NV3041A been added to TFT_eSPI yet? Tried using Squareline Studio with that display today and found out that Squareline Studio sadly seemingly relies on TFT_eSPI library to function with displays. If any here recommend a solution or workaround, thanks in advance. — Reply to this email directly, view it on GitHub <[#2579 (comment)](https://github.com/Bodmer/TFT_eSPI/discussions/2579#discussioncomment-10231196)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARYQN2Z4R5CJUTSAVLOSZR3ZPTYDLAVCNFSM6AAAAAAW2XCPO2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRTGEYTSNQ> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Oh? How to make Squareline Studio use Arduino_GFX library instead of TFT_eSPI?

Thanks in advance for explaining how to get that running properly on one of those displays! (Been trying for hours)

**Comment by CoolmdXi** [comment]
It's also supported in bb_spi_lcd display library

**Reply by 01GOD** [reply]
I have used that before in some of the CYD examples. But HOW to replace TFT_eSPI with that or Arduino_GFX in Squareline Studio projects so they compile and display properly? 

Thanks in advance!

**Reply by CoolmdXi** [reply]
I am afraid I dont know maybe Chrismendes will help you
